src/middleware/validation.middleware.ts

```diff
@@ -3,17 +3,18 @@ import { validationResult, FieldValidationError } from 'express-validator';
 
 import * as config from '../config';
 import { log } from '../utils/logger';
+import { validateFilepath } from '../utils/validateFilepath';
 import { FormattedValidationErrors } from '../model/validation.model';
 
 export const checkValidations = (req: Request, res: Response, next: NextFunction) => {
     try {
         const errorList = validationResult(req);
+        const validatedFilepath = validateFilepath(req, res);
```

Comment:
After the `validateFilepath` method changes we can move this line just after checking the error list on line 14.